textattack/transformations/word_swap_extend.py

```diff
@@ -0,0 +1,107 @@
+
+    contraction_map = {
+        "ain't": "isn't",
+        "aren't": "are not",
+        "can't": "cannot",
+        "can't've": "cannot have",
+        "could've": "could have",
+        "couldn't": "could not",
+        "didn't": "did not",
+        "doesn't": "does not",
+        "don't": "do not",
+        "hadn't": "had not",
+        "hasn't": "has not",
+        "haven't": "have not",
+        "he'd": "he would",
+        "he'd've": "he would have",
+        "he'll": "he will",
+        "he's": "he is",
+        "how'd": "how did",
+        "how'd'y": "how do you",
+        "how'll": "how will",
+        "how's": "how is",
+        "I'd": "I would",
+        "I'll": "I will",
+        "I'm": "I am",
+        "I've": "I have",
+        "i'd": "i would",
+        "i'll": "i will",
+        "i'm": "i am",
+        "i've": "i have",
+        "isn't": "is not",
+        "it'd": "it would",
+        "it'll": "it will",
+        "it's": "it is",
+        "ma'am": "madam",
+        "might've": "might have",
+        "mightn't": "might not",
+        "must've": "must have",
+        "mustn't": "must not",
+        "needn't": "need not",
+        "oughtn't": "ought not",
+        "shan't": "shall not",
+        "she'd": "she would",
+        "she'll": "she will",
+        "she's": "she is",
+        "should've": "should have",
+        "shouldn't": "should not",
+        "that'd": "that would",
+        "that's": "that is",
+        "there'd": "there would",
+        "there's": "there is",
+        "they'd": "they would",
+        "they'll": "they will",
+        "they're": "they are",
+        "they've": "they have",
+        "wasn't": "was not",
+        "we'd": "we would",
+        "we'll": "we will",
+        "we're": "we are",
+        "we've": "we have",
+        "weren't": "were not",
+        "what're": "what are",
+        "what's": "what is",
+        "when's": "when is",
+        "where'd": "where did",
+        "where's": "where is",
+        "where've": "where have",
+        "who'll": "who will",
+        "who's": "who is",
+        "who've": "who have",
+        "why's": "why is",
+        "won't": "will not",
+        "would've": "would have",
+        "wouldn't": "would not",
+        "you'd": "you would",
+        "you'd've": "you would have",
+        "you'll": "you will",
+        "you're": "you are",
+        "you've": "you have",
+    }
+
+    def _get_transformations(self, current_text, indices_to_modify):
+        """Return all possible transformed sentences, each with one
+        extension."""
+        transformed_texts = []
+        words = current_text.words
+        for idx in indices_to_modify:
+            word = words[idx]
+            # expend when word in map
+            if word in self.contraction_map:
+                expanded = self.contraction_map[word].split()
+                print(word, expanded)
+                transformed_text = current_text.replace_word_at_index(idx, expanded[0])
```

Comment:
I think you should just be able to do:
```
transformed_text = current_text.replace_word_at_index(idx, expanded)
```

and remove the calls to `insert_text_after_word_index`. `AttackedText` should be able to handle multi-word replacements like this; let me know if I'm wrong.